### PR TITLE
Use local images of BMO&CAMP3 for CI

### PR DESCRIPTION
**Pull request by fmuyassarov**
Add ability to use local images of BMO & CAMP3 when CI is triggered from Metal3-dev-env repo or as master job.

### CODE REVIEW

File: jenkins/scripts/files/run_integration_tests.sh [hunk 1]
```diff
@@ -45,8 +46,10 @@ fi
 
 if [ "${REPO_NAME}" == "metal3-dev-env" ]
 then
-   METAL3REPO="${UPDATED_REPO}"
-   METAL3BRANCH="${UPDATED_BRANCH}"
+   export METAL3REPO="${UPDATED_REPO}"
+   export METAL3BRANCH="${UPDATED_BRANCH}"
+   export BAREMETAL_OPERATOR_LOCAL_IMAGE="${M3PATH}/baremetal-operator"
+   export CAPM3_LOCAL_IMAGE="${M3PATH}/cluster-api-provider-metal3"
```
Comment (maelk) [inline jenkins/scripts/files/run_integration_tests.sh:52]: If you give it this way, they have to exist beforehand. The other way is to give the url to the github repo

Comment (fmuyassarov) [reply jenkins/scripts/files/run_integration_tests.sh]: Fixed

File: jenkins/scripts/files/run_integration_tests.sh [hunk 2]
```diff
@@ -45,8 +45,10 @@ fi
 
 if [ "${REPO_NAME}" == "metal3-dev-env" ]
 then
-   METAL3REPO="${UPDATED_REPO}"
-   METAL3BRANCH="${UPDATED_BRANCH}"
+   export METAL3REPO="${UPDATED_REPO}"
+   export METAL3BRANCH="${UPDATED_BRANCH}"
+   export BAREMETAL_OPERATOR_LOCAL_IMAGE="https://github.com/metal3-io/baremetal-operator.git"
+   export CAPM3_LOCAL_IMAGE="https://github.com/metal3-io/cluster-api-provider-metal3.git"
```
Comment (maelk) [inline jenkins/scripts/files/run_integration_tests.sh:51]: You should take the branch into account. if you are testing with v1alpha3 or v1alpha4, it is different branches in CAPM3 for example

File: jenkins/scripts/files/run_integration_tests.sh [hunk 3]
```diff
@@ -68,6 +70,10 @@ then
    export CAPM3BRANCH="${UPDATED_BRANCH}"
    export CAPM3PATH="/home/${USER}/tested_repo"
    export CAPM3_LOCAL_IMAGE="/home/${USER}/tested_repo"
+elif [[ "${REPO_NAME}" == "project-infra" ]]
+then
+   export BAREMETAL_OPERATOR_LOCAL_IMAGE="https://github.com/metal3-io/baremetal-operator.git"
+   export CAPM3_LOCAL_IMAGE="https://github.com/metal3-io/cluster-api-provider-metal3.git"
```
Comment (maelk) [inline jenkins/scripts/files/run_integration_tests.sh:76]: same here

File: jenkins/scripts/files/run_integration_tests.sh (hunk 2 above)
Comment (fmuyassarov) [reply jenkins/scripts/files/run_integration_tests.sh:51]: Fixed, thank you.

File: jenkins/scripts/files/run_integration_tests.sh (hunk 3 above)
Comment (fmuyassarov) [reply jenkins/scripts/files/run_integration_tests.sh:82]: Fixed.

**Comment by fmuyassarov** [comment]
/hold 
until [#304](https://github.com/metal3-io/metal3-dev-env/pull/304) gets merged.

**Pull request merged**